fuzz/Cargo.toml

```diff
@@ -60,3 +60,6 @@ doc = false
 debug = true
 debug-assertions = true
 overflow-checks = true
+#lto = true
```

Comment:
Can you say why these config options were added to `fuzz/Cargo.toml`?

Also, the commented-out line should be removed.

Reply:
The options are there to spend a bit more time optimizing the binary to hopefully get to run more fuzzing I think I forgot to remove this since it's not related to the topic